### ISSUE TITLE
 Protege doesn't handle a unionOf datatypes in gist:numericValue correctly

### ISSUE THREAD

**Issue by rjyounes** [semanticarts/gist#552]
The definition of `gist:numericValue` is:

```
gist:numericValue
	a owl:DatatypeProperty ;
	rdfs:domain gist:Magnitude ;
	rdfs:range [
		owl:unionOf (
			owl:rational
			owl:real
			xsd:byte
			xsd:decimal
			xsd:double
			xsd:float
			xsd:int
			xsd:integer
			xsd:long
			xsd:negativeInteger
			xsd:nonNegativeInteger
			xsd:nonPositiveInteger
			xsd:positiveInteger
			xsd:short
			xsd:unsignedByte
			xsd:unsignedInt
			xsd:unsignedLong
			xsd:unsignedShort
		) ;
	] ;
	skos:definition "The actual value of a magnitude."^^xsd:string ;
	skos:prefLabel "numeric value"^^xsd:string ;
	.
```

This is valid OWL (the range of `owl:unionOf` is `rdf:List`). However, Protege doesn't recognize it and:
* Considers all the datatypes as classes as well as datatypes
* Considers the property as both a datatype and object property

This is unfortunate because many of our users view the ontology in Protege. The only stopgap I can think of is to add a `skos:note` to the property definition explaining the issue. Is there any other approach? Is there any way to change the axiom in a way that Protege accepts?

**Comment by sa-bpelakh** [comment]
Is this a known issue we can track? Or perhaps report to https://github.com/protegeproject/protege? It seems to be actively maintained.

Regardless, as I suggested on the pull request, is this important enough that we lose the `rdfs:range` and go to `gist:rangeIncludes` instead, which loses us the reasoning but makes Protege happy?

**Comment by rjyounes** [comment]
I don't think we should step back to `gist:rangeIncludes`. It's too weak in this case, where we actually know the values and are not just suggesting them.

**Comment by rjyounes** [comment]
So my suggestion is to document it in the property definition with a `skos:note` - and possibly elsewhere if there's somewhere that would get more visibility. Submitting the issue to the protege project is a good idea, but independent of the action we take here on gist.

**Comment by rjyounes** [comment]
Protégé issue: https://github.com/protegeproject/protege/issues/1029.

**Comment by rjyounes** [comment]
@marksem's comment on PR #535:

Apparently Protege is ok with something like this:

```
<http://ns/myprop1> rdf:type owl:DatatypeProperty ;
                    rdfs:range [ rdf:type rdfs:Datatype ;
                                 owl:unionOf ( owl:rational
                                               owl:real
                                             )
                               ] .

```

So we're just missing the `rdf:type rdfs:Datatype` part and then we're all good.

**Comment by rjyounes** [comment]
@sa-bpelakh You are off the hook. @marksem or I will take care of this from here.

**Comment by marksem** [comment]
Looks like we omitted something the OWL 2 spec says we need: the declaration `rdf:type rdfs:Datatype`

Per the spec:

7.2 Union of Data Ranges
A union data range DataUnionOf( DR1 ... DRn ) contains all tuples of literals that are contained in the at least one data range DRi for 1 ≤ i ≤ n. All data ranges DRi must be of the same arity, and the resulting data range is of that arity as well.

DataUnionOf := 'DataUnionOf' '(' DataRange DataRange { DataRange } ')'

The following data range contains all strings and all integers:

Functional-Style Syntax:
DataUnionOf( xsd:string xsd:integer )
RDF:
_:x rdf:type rdfs:Datatype .
_:x owl:unionOf ( xsd:string xsd:integer ) .

**Comment by rjyounes** [comment]
Fixed in PR #553.
